core/src/main/java/org/opensearch/sql/planner/logical/LogicalRelation.java

```diff
@@ -25,13 +26,18 @@ public class LogicalRelation extends LogicalPlan {
   @Getter
   private final Table table;
 
+  @Getter
+  @Setter
+  private Integer pageSize;
```

Comment:
what is the meaning of pagesize in logcial relation. does LogicalLimit already expression this?

Reply:
`LogicalLimit` reflects `LIMIT` clause and it is not related to page size. For example, query `SELECT * FROM <index> LIMIT 1000` with `fetch_size = 20` should return 51 pages: 50 pages with data and last empty page.